<?xml version="1.0" encoding="utf-8"?>

<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
  <!-- ls -ltr /dev/video* 라인으로 확인하기 -->
    <param name="video_device" value="/dev/video2" />
    <param name="framerate" value="30" />
    <!-- <param name="image_width" value="1920" />
    <param name="image_height" value="1080" />     -->
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />    
    <!-- <param name="image_width" value="640" />
    <param name="image_height" value="480" />     -->
    <!-- <param name="pixel_format" value="yuyv" /> -->
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node>
</launch>
